### DISCUSSION TITLE
Beta Cloning

### DISCUSSION THREAD

**Discussion by sandrojclic** [erew123/alltalk_tts#315] [Dev Discussions]
Hello. I'm at https://github.com/erew123/alltalk_tts/tree/alltalkbeta. From the green dropdown button, do I get version 1 or version 2? I want to use GitHub Desktop for cloning. Thank you.

**Comment by erew123** [comment]
Hi @sandrojclic I dont use github desktop, however, anything cloned from https://github.com/erew123/alltalk_tts/tree/alltalkbeta is v2

Manual instructions for cloning are on the quick setup https://github.com/erew123/alltalk_tts/tree/alltalkbeta?tab=readme-ov-file#-quick-setup-text-generation-webui--standalone-installation

**Reply by sandrojclic** [reply]
Thank you very much for your kind response and congratulations on this wonderful software.
